tests/io/test_api.py

```diff
@@ -13,31 +13,37 @@
 
 @mark.api
 def test_load_start_only():
-    data = aeon.load(nonmonotonic_path, exp02.Patch2.Encoder, start=pd.Timestamp("2022-06-06T13:00:49"))
+    data = aeon.load(
+        nonmonotonic_path, exp02.Patch2.Encoder, start=pd.Timestamp("2022-06-06T13:00:49"), downsample=None
+    )
```

Comment:
Setting `downsample=None` has the effect of just loading the raw data. We do not need or care about validating the frequency of the data since we are simply interested in analyzing the behavior of load in this specific dataset. The reason we had to introduce this parameter is that here we opted to set the default to `downsample=True`.

This is a way to ensure the behavior of this test does not change from the previous version.